ansible/roles/stack-sunbird/templates/sunbird_groups-service.env

```diff
@@ -18,6 +18,10 @@ sunbird_sso_url={{keycloak_auth_server_url}}/
 sunbird_sso_realm={{keycloak_realm}}
 sunbird_sso_client_id={{sunbird_sso_client_id}}
 sunbird_sso_client_secret={{sunbird_sso_client_secret | d('')}}
+sunbird_sso_username={{sunbird_sso_username}}
+sunbird_sso_password={{sunbird_sso_password}}
+sunbird_keycloak_user_federation_provider_id={{core_vault_sunbird_keycloak_user_federation_provider_id}}
+sunbird_keycloak_required_action_link_expiration_seconds={{sunbird_keycloak_required_action_link_expiration_seconds}}
```

Comment:
@Hari-stackroute where is the default value for this ?

Reply:
Defaults are in all.yml